spec/models/post_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject do
+    Post.new(
+      author: User.new(name: 'Tom', photo: 'my_photo.com', bio: 'Bio message'),
+      title: 'post title',
+      text: 'post text',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+  before { subject.save }
+
+  it 'title should be present' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'There can be no more than 250 characters in the title' do
+    long_title = 'Now we will create it blocks for each behaviour were
+    expecting in the Movies model. So remember to be as clear as possible
+    when describe your tests because you need to be a good communicator and
+    make yourself understandable not only with computers but also with people.'
+    subject.title = long_title
+    expect(subject).to_not be_valid
+  end
+
+  it 'comments_counter should be positive' do
+    subject.comments_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'likes_counter should be positive' do
+    subject.likes_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'shows at most 5 recent comments' do
+    subject.comments = [
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!'),
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!'),
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!'),
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!'),
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!'),
+      Comment.new(post: subject, author: subject.author, text: 'Hi Tom!')
+    ]
+    expect(subject.recent_comments.length).to eql(5)
+  end
```

Comment:
- [ ] Great job adding tests to user and post models, as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Kindly add a test case for `update_post_counter` method too.

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please also add tests to `comment` and `like` models too.